Server/src/bgp/linkstate/MPLinkStateAttr.cpp

```diff
@@ -57,6 +57,21 @@ namespace bgp_msg {
         }
     }
 
+    float MPLinkStateAttr::float_swap(float value)
```

Comment:
Can you update this to support IEEE754 float?  The parsed value is currently the ieee_float,  we want to translate that to KBits per second.